debugging-sos-lldb-via-core/test.sh

```diff
@@ -32,8 +32,10 @@ sdk_version=$1
 
 set -x
 
-dotnet tool install -g dotnet-sos  --version 3.0.0-preview8 || true
-dotnet tool install -g dotnet-dump  --version 3.0.0-preview8 || true
+dotnet tool uninstall -g dotnet-sos || true
+dotnet tool uninstall -g dotnet-dump || true
+dotnet tool install -g dotnet-sos  --version 3.0.0
+dotnet tool install -g dotnet-dump --version 3.0.0
```

Comment:
I wonder what happens when 3.1 is released. Anyway, that's a question for another time.

Reply:
Maybe we shouldn't specify a version here, to check whether the latest version of the tool works?

Reply:
That sounds like a great idea!